src/main/java/n64loaderwv/DPDLL.java

```diff
@@ -42,65 +45,53 @@
 import ghidra.util.exception.InvalidInputException;
 import ghidra.util.task.TaskMonitor;
 
+import n64loaderwv.Utils;
+
 public class DPDLL 
 {
 	private Program program;
 	public int dll_id;
 	public int dll_rom_offset;
 	public int dll_tab_offset;
+	public int dll_bss;
 	public int dll_size;
 	public int hdr_size;
-	public int hdr_offset_data2;
-	public int hdr_offset_constants; // includes import/export function table
-	public short hdr_unk1;
+	public int hdr_offset_data;
+	public int hdr_offset_rodata; // begins with relocation table
+	public short hdr_export_count;
 	public int code_offset;
 	public int code_size;
-	public int constants_offset;
-	private long dll_address;
+	public int rodata_offset;
+	private long load_address;
 	private long code_address;
+	private long rodata_address;
+	private long data_address;
 	private AddressSpace address_space;
-	private long load_address;
 	public String dll_identifier;
 	public String dll_block_name;
-	
-	private long MakeRomAddress(int romOffset)
-	{
-		return load_address + (romOffset - 0x1000);
-	}
-	
-	private int GetHeaderExportFuncOffset(int idx)
+
+	private int GetHeaderInitFuncOffset()
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.